code/datums/movement/mob.dm

```diff
@@ -14,11 +14,18 @@
 
 // Death handling
 /datum/movement_handler/mob/death/DoMove()
+	if(mob.moved_by_grab)
```

Comment:
The appropriate solution is:
```
/datum/movement_handler/mob/death/DoMove(var/direction, var/mob/mover)
    if(mob != mover)
        return MOVEMENT_HANDLED
    [...]
```
No other changes should be needed

Reply:
thanks. This what I was looking for